src/test/scala/iog/psg/cardano/util/ModelCompare.scala

```diff
@@ -0,0 +1,136 @@
+package iog.psg.cardano.util
+
+import iog.psg.cardano.CardanoApiCodec._
+import org.scalatest.Assertion
+import org.scalatest.matchers.should.Matchers
+
+trait ModelCompare extends Matchers {
+
+  final def compareInAddress(decoded: InAddress, proper: InAddress): Assertion = {
```

Comment:
Can you use 'expected' instead of 'proper'?

Reply:
yes sure